packages/core/admin/ee/admin/pages/SettingsPage/SingleSignOn/index.js

```diff
@@ -100,8 +101,14 @@ export const SingleSignOn = () => {
             <LoadingIndicatorPage />
           ) : (
             <Layout>
-              <Stack size={12}>
-                <Box background="neutral0" padding={6} shadow="filterShadow" hasRadius>
+              <Box background="neutral0" padding={6} shadow="filterShadow" hasRadius>
+                <Stack size={4}>
```

Comment:
Since the Stack component is already a Box, maybe you can try that:

```js
<Stack size={4} background="neutral0" padding={6} shadow="filterShadow" hasRadius>
   ...
</Stack>
```

Reply:
indeed thanks, and the Layout component was misplaced I fixed it too!